src/System.Management.Automation/namespaces/LocationGlobber.cs

```diff
@@ -1368,24 +1359,16 @@ internal static bool IsProviderQualifiedPath(string path)
         /// <exception cref="ArgumentNullException">
         /// If <paramref name="path"/> is null.
         /// </exception>
-        internal static bool IsProviderQualifiedPath(string path, out string providerId)
+        internal static bool IsProviderQualifiedPath(ReadOnlySpan<char> path, out ReadOnlySpan<char> providerId)
         {
-            // Verify parameters
-
-            if (path == null)
-            {
-                throw PSTraceSource.NewArgumentNullException(nameof(path));
-            }
-
-            providerId = null;
+            providerId = ReadOnlySpan<char>.Empty;
```

Comment:
If we're OK with removing the argument check, we should also remove the exception note in the doc comment.

Reply:
Done.